modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerm.scala

```diff
@@ -157,8 +158,19 @@ case class RouteMeta(path: String, method: HttpMethod, operation: Operation) {
       })
 }
 
+sealed trait SecurityScheme {
+  def typeName: Option[String]
+  def extensions: Map[String, AnyRef]
```

Comment:
Is there a well-typed variant that we could use instead of `AnyRef`? Refactors or API changes that would need to be changed in more than one place would be more difficult to find if we use reflection all over the place

Reply:
This is just a reflection of what the swagger parser gives us: `java.util.Map[String, Object]`.  There isn't really a better way to represent it since keys could be anything supported in yaml.

Reply:
What are they consumed by? Something is going to need to do reflection, just wondering if there's some sort of schema we can apply earlier in the process

Reply:
Soooo, nothing, it turns out.  I was originally going to support non-standard security types via vendor extensions on the `SecurityScheme` instance, but I ended up finding it's easier/nicer to just (ab)use the auth-scheme field when the security type is `http`.  So I can just remove this field entirely.

Reply:
Sick. We can explicitly add vendor extensions as necessary.